src/Bot.js

```diff
@@ -1,35 +1,49 @@
 class Bot {
+
+    /**
+     * @param  {Number} waitingTime Time in seconds to wait before reviving
+     */
     constructor(waitingTime) {
         this.channels = new Map();
-        this.waitingTime = waitingTime;
+        this.waitingTime = waitingTime * 1000;
```

Comment:
Times 1000? It's still using getTime, wouldn't this make it at least 1000s timeout?

Reply:
I'm not exactly sure why, but it was giving me ms values from getTime.

Reply:
Just checked and ms is the spec. I'm not sure why seconds was working before

Reply:
Woah weirddd!!